src/main/java/com/checkout/common/PaymentSource.java

```diff
@@ -0,0 +1,38 @@
+package com.checkout.common;
+
+import com.google.gson.annotations.SerializedName;
+
+public enum PaymentSource {
+
+    @SerializedName("card")
+    CARD,
+    @SerializedName("id")
+    ID,
+    @SerializedName("customer")
+    CUSTOMER,
+    @SerializedName("dlocal")
+    DLOCAL,
+    @SerializedName("token")
+    TOKEN,
+    @SerializedName("baloto")
+    BALOTO,
+    @SerializedName("boleto")
+    BOLETO,
+    @SerializedName("fawry")
+    FAWRY,
+    @SerializedName("giropay")
+    GIROPAY,
+    @SerializedName("ideal")
+    IDEAL,
+    @SerializedName("oxxo")
+    OXXO,
+    @SerializedName("pagofacil")
+    PAGO_FACIL,
+    @SerializedName("rapipago")
+    RAPI_PAGO,
+    @SerializedName("klarna")
+    KLARNA,
+    @SerializedName("id")
```

Comment:
Please confirm if `id` is the correct code for `SEPA`.

Reply:
Yes it is